ddtrace/tracer.py

```diff
@@ -13,7 +13,6 @@
 from .span import Span
 from .utils.formats import get_env
 from .utils.deprecation import deprecated
-from .utils.runtime import generate_runtime_id
```

Comment:
I guess you can delete this function from utils?

Reply:
I did, just deleted the whole file.

Reply:
🤦‍♂